tools/maven-plugin/src/main/java/io/smallrye/openapi/mavenplugin/GenerateSchemaMojo.java

```diff
@@ -419,6 +432,7 @@ private Map<String, String> getProperties() throws IOException {
         addToPropertyMap(cp, OpenApiConstants.SCAN_PROFILES, scanProfiles);
         addToPropertyMap(cp, OpenApiConstants.SCAN_EXCLUDE_PROFILES, scanExcludeProfiles);
         addToPropertyMap(cp, OpenApiConstants.SCAN_RESOURCE_CLASS_PREFIX, scanResourceClasses);
+        addToPropertyMap(cp, OpenApiConstants.INFO_TITLE, infoTitle);
```

Comment:
Is this intended? Same as line 422.

Reply:
:man_facepalming: Definitely not, a copy/paste leftover of mine.
Thanks for catching it, should be fixed now